docs/UserGuide/Ecosystem Integration/Zeppelin-IoTDB.md

```diff
@@ -0,0 +1,184 @@
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+    
+        http://www.apache.org/licenses/LICENSE-2.0
+    
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+
+# 1. Outline
+
+- What's Zeppelin
+
+- IoTDB Zeppelin-Interpreter 
+  - System Requirements
+  - Build Interpreter
+  - Install Interpreter
+  - Running Zeppelin and IoTDB
+- Use Zeppelin-IoTDB
+- Configuration
+
+
+
+# 2. What's Zeppelin
+
+Zeppelin is a web-based notebook that enables interactive data analytics. You can connect to data sources and perform interactive operations with SQL, Scala, etc. The operations can be saved as documents, just like Jupyter. Zeppelin has already supported many data sources, including Spark, ElasticSearch, Cassandra, and InfluxDB. Now, we have enabled Zeppelin to operate IoTDB via SQL. 
+
+![image-iotdb-note-snapshot](../../img/UserGuide/EcosystemIntegration/iotdb-note-snapshot.png)
```

Comment:
Actually we don't have to upload the image to our repository, you may upload the image to https://github.com/thulab/iotdb/issues/543 for the url and use that url in the docs.

Reply:
Thanks